ui/src/containers/Winner/index.tsx

```diff
@@ -22,12 +28,12 @@ const Winner = ({}) => {
         <img className={style.winner_gif_container} src={gif} alt='' />
         <div className={style.result}>
           <div className={style.result_award}>
-            <span>$10,000</span>
+            <div>{priceText && (priceText.length > 20 ? priceText.slice(0, 20) + '...' : priceText)}</div>
```

Comment:
I suggest to use css line clamp to replace this, check the article here

https://css-tricks.com/almanac/properties/l/line-clamp/

Reply:
is it need to add with`-webkit-` prefix ,and need to changing the display attribute?
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp